quickjs.c

```diff
@@ -1388,6 +1388,12 @@ void *js_realloc_rt(JSRuntime *rt, void *ptr, size_t size)
     return rt->mf.js_realloc(&rt->malloc_state, ptr, size);
 }
 
+static void *js_realloc_opaque(void *opaque, void *ptr, size_t size)
```

Comment:
Oh I forgot to hit send on my 2nd comment, sorry. Can you please rename this to, say, `js_dbuf_realloc` ?

Reply:
No worries! Done